src/BinSkim.Driver/AnalyzeCommand.cs

```diff
@@ -68,6 +70,8 @@ public override int Run(AnalyzeOptions analyzeOptions)
 
             int result = base.Run(analyzeOptions);
 
+            CompilerDataLogger.Flush();
```

Comment:
>Flush [](http://example.com/codeflow?start=31&length=5)

If a flush is really required should this be in a finally block somewhere? This won't execute in a case where Run raises an exception

Reply:
yeah, AppInsights batches the metrics, so we will need to flush.
Added a try/finally.